handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -1143,7 +1209,13 @@ protected static TrustManagerFactory buildTrustManagerFactory(
     static TrustManagerFactory buildTrustManagerFactory(
             X509Certificate[] certCollection, TrustManagerFactory trustManagerFactory)
             throws NoSuchAlgorithmException, CertificateException, KeyStoreException, IOException {
-        final KeyStore ks = KeyStore.getInstance(KeyStore.getDefaultType());
+        return buildTrustManagerFactory(certCollection, trustManagerFactory, KeyStore.getDefaultType());
+    }
+
+    static TrustManagerFactory buildTrustManagerFactory(
```

Comment:
Instead of creating a new function, just modify the old one, since it was package-private. Unless there's a lot a usages.